literature_search.py

```diff
@@ -0,0 +1,298 @@
+#==============================================================================
+#=========================  Define the search criteria ========================
+
+from Bio import Entrez
+import numpy as np
+
+def search(query):
+    Entrez.email = 'your.email@example.com'
+    handle = Entrez.esearch(db='pubmed', 
+                            sort='most recent', 
+                            retmax='5000',
+                            retmode='xml', 
+                            datetype = 'pdat',
+                            reldate = 7, #only within n days from now
+#                             mindate = '2019/03/25',
+#                             maxdate = '2019/03/27', #for searching date range
+                            term=query)
+    results = Entrez.read(handle)
+    return results
+
+#search terms (can test string with Pubmed Advanced Search)
+search_results = search('(Biomech*[Title/Abstract] OR locomot*[Title/Abstract])')
+
+#==============================================================================
+#========================= Perform Search and Save Paper Titles ===============
+def fetch_details(id_list):
+    Entrez.email = 'your.email@example.com'
+    handle = Entrez.efetch(db='pubmed',
+                           retmode='xml',
+                           id=id_list)
+    results = Entrez.read(handle)
+    return results
+
+id_list = search_results['IdList']
+papers = fetch_details(id_list)
+print("")
+
+# Definitely could change these loops for speed.
```

Comment:
Try something like this to avoid loops for speed:
````
titles = [0]*len(papers['PubmedArticle'])
#works:
a = [4,4,4,4] #create list
b = [0]*len(a) #initialize without loop (numeric)
b2 = ['']*len(a) #initialize without loop (string)
````

Reply:
I doubt this will ever need to be changed for speed, like if we had a million billion papers it might, but it won't.

I'll array it, though it might not actually be faster.

Reply:
Just responding to your code comments, Gary! It's midnight here. you're lucky I'm staying up for GoT.